planemo/shed_lint.py

```diff
@@ -50,6 +52,21 @@
 ]
 
 
+# TODO: put this back into tool_util, my bad :(
```

Comment:
So, this should be moved back to galaxy?

Reply:
Yep, I think we need to maintain the interface

Reply:
Yes, we can backport.

Reply:
OK. Done https://github.com/galaxyproject/galaxy/pull/15457